npm/test/upload-form-files.test.ts

```diff
@@ -131,7 +153,10 @@ test("dropzone updates the page with name of file if a .docx file has been dropp
   const dragEvent = new dragEventClass()
   await mockDom.form.handleDroppedItems(dragEvent)
 
-  verifyVisibilityOfSuccessAndRemovalMessage(mockDom.successAndRemovalMessageContainer!, true)
+  verifyVisibilityOfSuccessAndRemovalMessage(
+    mockDom.successAndRemovalMessageContainer!,
+    true
+  )
```

Comment:
I don't know the reason to have the statement in multi-line, I'll write in a single line only because it will increase the number of lines in the file.

Reply:
Ah 😄 . Prettier (the formatter) did this

Reply:
hmm ok